Tests/AppTests/__Snapshots__/WebpageSnapshotTests/test_KeywordShow.1.txt

```diff
@@ -133,6 +133,7 @@
         <p>
           <strong>10 packages.</strong>
         </p>
+        <ul class="pagination"></ul>
```

Comment:
We'll also need to update the image snapshot before merging

Reply:
I just updated the snapshots and there's no image diff with this change to the HTML. The `ul` would only render visually if there were a `li` inside it.

Maybe the test should have more test data so it shows actual pagination controls?

Reply:
I've just had a look but unless we also fiddle with the snapshot size it's not going to show the page links:

<img width="626" alt="CleanShot 2021-07-08 at 10 20 40@2x" src="https://user-images.githubusercontent.com/65520/124888480-6e369b80-dfd6-11eb-9647-9da9f4ca7679.png">

I'd say we just leave it at that, ok?

Reply:
Absolutely!